pgbouncer/tests/test_pgbouncer_integration_e2e.py

```diff
@@ -32,29 +31,28 @@ def test_check(instance, aggregator, datadog_agent):
     check.check_id = 'test:123'
     check.check(instance)
 
-    version = _get_version_from_env()
+    version = common.get_version_from_env()
     assert_metric_coverage(version, aggregator)
-    patch = '1' if version[1] == '8' else '2'
+    patches = {'8': '1', '9': '2'}
+    patch = patches.get(version[1])
 
     version_metadata = {
-        'version.raw': '.'.join(version) + '.' + patch,
+        'version.raw': '.'.join(version),
         'version.scheme': 'semver',
         'version.major': version[0],
         'version.minor': version[1],
-        'version.patch': patch,
     }
+    if patch:
+        version_metadata['version.patch'] = patch
+        version_metadata['version.raw'] += '.' + patch
```

Comment:
(nit) I think it might be cleaner if we use two different tags for the docker image and version to avoid having to fix expected versions in tests:

https://github.com/DataDog/integrations-core/blob/666f871e6dc6f42d68aa5b4b954f96813de0d789/fluentd/tox.ini#L20-L21